regression/cbmc/gcc_bswap1/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
```

Comment:
Having one positive test does not seem enough to show that it works (a contradiction in the solver would give the same result).
We should have also some tests for which verification fail (for instance bswap a particular value and assert it is different from the expected result).